modin/pandas/test/dataframe/test_udf.py

```diff
@@ -338,6 +338,18 @@ def test_eval_df_arithmetic_subexpression():
     df_equals(modin_df, df)
 
 
+def test_eval_groupby_transform():
+    # see #5511 for details
+    df = pd.DataFrame({"num": range(1, 1001), "group": ["A"] * 500 + ["B"] * 500})
+    assert df.groupby("group")["num"].transform("min").unique().tolist() == [1, 501]
```

Comment:
this isn't about `eval()` with a scalar, now is it? 😜

Reply:
Will it be enough to correct the name of the pull request or do you suggest splitting it?